tox.ini

```diff
@@ -14,7 +14,7 @@ deps=
 extras=
     dataflow_graphic
 # Set the environment variable INMANTA_EXTRA_PYTEST_ARGS='--fast' to run in fast mode
-commands=timeout -s SIGINT {env:INMANTA_TIMEOUT_TESTSUITE:50}m py.test --log-level DEBUG --full-trace --cov=inmanta --cov-report term --cov-report xml --cov-config=pyproject.toml --junitxml=junit-{envname}.xml -vvv {env:INMANTA_EXTRA_PYTEST_ARGS:} --durations={env:INMANTA_TIMEOUT_TESTSUITE:50} tests
+commands=timeout -s SIGINT {env:INMANTA_TIMEOUT_TESTSUITE:50}m py.test --log-level DEBUG --full-trace -p no:sugar --cov=inmanta --cov-report term --cov-report xml --cov-config=pyproject.toml --junitxml=junit-{envname}.xml -vvv {env:INMANTA_EXTRA_PYTEST_ARGS:} --durations={env:INMANTA_TIMEOUT_TESTSUITE:50} tests
```

Comment:
Should we remove pytest-sugar from the dependencies?

Reply:
I would ask this on slack